client/taskenv/env.go

```diff
@@ -620,10 +624,12 @@ func (b *Builder) buildEnv(allocDir, localDir, secretsDir string,
 	// Build the Nomad Workload Token
 	if b.workloadTokenDefault != "" {
 		envMap[WorkloadToken] = b.workloadTokenDefault
+		envMap[UnixAddr] = "unix:" + filepath.Join(secretsDir, "api.sock")
```

Comment:
Aren't these usually of the form: `unix://$PATH`? ex. `unix:///secrets/api.sock`

Reply:
Right?? That's part of why this is all needed - the world is not consistent.  With a `caddy` reverse proxy, [for example](https://caddyserver.com/docs/caddyfile/directives/reverse_proxy#upstream-addresses), it's `unix//path/to/sock` :shrug: 

In our particular case, it can be either `unix:{path}` or `unix://{path}` (but _not_ `unix:/{path}`).

Would you prefer `unix://`?

Reply:
> the world is not consistent

Yeah, there's no real standard here. The Go parser (which we're using in the `api` package) handles both https://go.dev/play/p/S2tCmK6la8H I think I'd prefer `unix://` just because it matches the `file://` URL scheme (from RFC1738 and friends). But that's a matter of taste.